scripts/travis/update/update-project.sh

```diff
@@ -0,0 +1,132 @@
+!/usr/bin/env bash
+
+git config --global user.name "alfresco-build"
+git config --global user.email "alfresco-build@hyland.com"
+
+BUILD_PIPELINE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_DIR="$BUILD_PIPELINE_DIR/../.."
+
+TEMP_GENERATOR_DIR=".tmp-generator";
+BRANCH_TO_CREATE="update-alfresco-aca-dependencies"
+TOKEN=""
+PR_NUMBER=""
+DRY_RUN="false"
+
+show_help() {
+    echo "Usage: update-project.sh"
+    echo ""
+    echo "-t or --token: Github ouath token"
+    echo "-p or --pr: Originating ACA PR number"
+    echo "-v or --version version to update"
+    echo "-d or --dry-run: The script won't execute critical operation, just simulate them"
+}
+
+set_token() {
+    TOKEN=$1
+}
+
+set_pr() {
+    PR_NUMBER=$1
+}
+
+version() {
+    VERSION=$1
+}
+
+set_commit() {
+    COMMIT=$1
+}
+
+set_dryrun() {
+
+    DRY_RUN="true"
+
+}
+
+update_dependency() {
+    PKG=$1
+    PKG_VERSION=$(npm view $PKG@$VERSION version)
+    echo "Update $PKG to $PKG_VERSION in $NAME_REPO"
+
+    for i in $(find . ! -path "*/node_modules/*" -name "package-lock.json" | xargs grep -l $PKG); do
+        directory=$(dirname $i)
+        echo "Update $PKG in  $directory"
+        ( cd $directory ; npm i --ignore-scripts $PKG@$PKG_VERSION --save-exact)
+    done
+
+    git add .
+    git commit -n -m "[ci:force][auto-commit] Update $PKG to $PKG_VERSION for branch: $BRANCH_TO_CREATE originated from $PKG PR: $PR_NUMBER"
+}
+
+update() {
+    NAME_REPO=$1
+    PKG_VERSION=$(npm view $PKG@$VERSION version)
+    echo "Update dependencies $NAME_REPO"
+
+    git clone https://$TOKEN@github.com/Alfresco/$NAME_REPO.git $TEMP_GENERATOR_DIR
+    cd $TEMP_GENERATOR_DIR
+
+    git fetch
+
+    # Checkout branch if exist, otherwise create it
+    BRANCH_CREATED=false
+    if git checkout $BRANCH_TO_CREATE 2>/dev/null ; then
+        git reset --hard origin/develop
+    else
+        BRANCH_CREATED=true
+        git checkout -b $BRANCH_TO_CREATE origin/develop
+    fi
+
+    update_dependency "@alfresco/aca-shared"
+    update_dependency "@alfresco/aca-content"
+    update_dependency "@alfresco/aca-about"
+    update_dependency "@alfresco/aca-preview"
+    update_dependency "@alfresco/aca-viewer"
+    update_dependency "@alfresco/aca-folder-rules"
```

Comment:
we are missing `@alfresco/aca-settings` and `@alfresco/adf-office-services-ext`

Reply:
Added